src/components/map/ha-location-editor.ts

```diff
@@ -77,7 +77,9 @@ class LocationEditor extends LitElement {
   }
 
   private _updateLocation(latlng: LatLng) {
-    this.location = this._ignoreFitToMap = [latlng.lat, latlng.lng];
+    // Normalize longitude if map provides values beyond -180 to +180 degrees.
+    const longitude = (((latlng.lng % 360.0) + 540.0) % 360.0) - 180.0;
```

Comment:
Can we only do this is the longitude is beyond -180 to +180?
```
if (Math.abs(latlng.lng) > 180) {
   ...
}
```

Reply:
OK, I'll add that.